docs/usecases/datastorage_for_institutions.rst

```diff
@@ -0,0 +1,314 @@
+face minimal technical overhead to interact with the data store.
+
+The infrastructural changes are accompanied by changes in the mindset and workflows
+of the researchers that perform analyses on the cluster.
+By using the data store, the institutes work routines are adjusted around
+DataLad datasets: Analyses are set-up inside of DataLad datasets, and for every
+analysis, an associated ``project`` is created under the namespace of the
+institute on the institute's :term:`Gitlab` instance automatically. This has
+the advantage of vastly simplified version control, even for trainees, and
+simplified access to projects for collaborators and supervisors. Data
+from the data store is installed as subdatasets. This comes with several
+benefits: Analyses are automatically linked to data, no unused file
+copies waste disk space on the compute cluster as data can be retrieved
+on-demand, and files that are easily re-obtained or recomputed can safely be
+dropped to save even more disk-space. More over, upon creation of an analysis
+project, the associated Gitlab project it is automatically configured as a remote
+with a publication dependency on the data store, thus enabling vastly simplified
+data publication routines and back-up of pristine results.
+
+
+Step-by-step
+^^^^^^^^^^^^
+
+.. note::
+
+   This use case describes the data storage implementation as done in INM-7,
+   research centre Juelich, Germany.
+
+To create a data store, parts of the old compute cluster and parts of the
+super computer at the Juelich supercomputing centre (JSC) are used to store
+large amounts of data. Thus, multiple different, independent machines take care of
+warehousing the data. While this is unconventional, it is convenient: The
+data does not strain the compute cluster, and with DataLad, it is irrelevant
+where the data is located.
+
+.. figure:: ../artwork/src/ephemeral_infra.svg
+   :alt: A simple, local version control workflow with datalad.
+   :figwidth: 80%
+
+   Trinity of research data handling: The data store (``$DATA``) is managed and
+   backed-up. The compute cluster (``$COMPUTE``) has an analysis-appropriate structure
+   with adequate resources, but just as users workstations/laptops (``$HOME``),
+   it is not concerned with data hosting.
+
+On their own machines (``$HOME``), researchers are free to do whatever they want
+as long as it is within the limits of their machines. The cluster (``$COMPUTE``)
+pulls the data exclusively from the data store (``$DATA``). Thus, within
+``$HOME``, researchers are free to explore data from ``$DATA`` as they wish,
+but scaling requires them to use ``$COMPUTE``. Results from ``$COMPUTE`` are pushed
+back to ``$DATA``, and hence anything that is relevant for a computation is tracked
+(and backed-up) there.
+
+The data store as a Git-annex RIA remote
+""""""""""""""""""""""""""""""""""""""""
+
+The distributed data store exists thanks to Git-annex: Any large file content in
```

Comment:
The distributed data store exists thanks to Git-annex (which DataLad builds upon):

Reply:
"Any large file content"; be careful about using the word "content". Git tracks content, but git-annex tracks files. I would say: "Any large file in a dataset...".

Though that's a bit of a trap, because it's configurable, etc.